source/plugin/entities/modifying.rst

```diff
@@ -0,0 +1,80 @@
+===================
+Modifying an Entity
+===================
+
+Sure, spawning a regular-old entity is nice and all, but there has to be something more interesting than that? This is
+where ``DataManipulator``\s come into play. An ``Entity`` is a ``DataHolder``, which means that our ``Entity`` can hold
+data. More on ``DataHolder``\s can be found in the :doc:`data documentation <../data/index>`.
+
+``DataManipulator``\s that apply to ``Entity``\s are things such as ``FoodData`` or ``HealthData``.
+A list of applicable ``DataManipulator``\s can be found at ``org.spongepowered.api.data.manipulator.mutable`` and
+``org.spongepowered.api.data.manipulator.mutable.entity``.
```

Comment:
Also in every subpackage of `org.spongepowered.api.data.manipulator.mutable.entity`. However is important to emphasize that not every `DataManipulator` in there is applicable for every `Entity`. That should be clarified.